APIcall.js

```diff
@@ -0,0 +1,13 @@
+const axios = require('axios');
+
+var app = express();
+
+app.get("/results", function(req,res) {
+    axios("https://api.openweathermap.org/data/2.5/weather?q=Seattle&appid=34e47a9c09f7eed541ec45ab96dda7e8", function(error, response, body) {
```

Comment:
Let's remove this for now :) While testing, I can use my own API key

Reply:
Sorry, by "this" I mean the API key haha

Reply:
Seconded, with bots that scan github for API keys going around and all. haha